mzLib/FlashLFQ/PeakIndexingEngine.cs

```diff
@@ -112,7 +121,6 @@ public void ClearIndex()
                     _indexedPeaks[i] = null;
                 }
             }
-
             GC.Collect();
```

Comment:
Forcing garbage collection is VERY dangerous for time to run algorithm. Is it necessary here?